scripts/script.yaml

```diff
@@ -379,8 +379,7 @@
     - name: exposures
       steps:
         - say: בקרבת כמה אנשים שהית בטווח 2 מטרים מעל ל-15 דקות ביממה האחרונה?
```

Comment:
this line is back to the problematic wording

Reply:
thought you were right, but this is what Mushon suggested, (looked at the commit).

Reply:
this is the old wording
you fixed it based on the comment that it's a question without an option to answer.

here:
https://github.com/hasadna/avid-covider/pull/58/commits/2b703d6d2d11563ddc4684a1785be926f1cf375d

Reply:
plus, when mushon suggested it there was a possible answer for this question:

https://github.com/hasadna/avid-covider/pull/56/commits/43041671d2758e7e9996636e2ec1a4c9351f68c5

Reply:
(I am not sure this is right, but must run. applied)